test/metric_value_benchmark/placeholder_test.go

```diff
@@ -12,31 +12,35 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
 )
 
-const dummyTestName = "Dummy"
-
 type DummyTestRunner struct {
 }
 
+var _ ITestRunner = (*DummyTestRunner)(nil)
```

Comment:
Didn't I delete the dummy test runner in the main repo?

Reply:
oh that is so weird, I cherry-picked your commit though. Guess it didn't delete for me.

Reply:
fixed